pkg/apis/storage/validation/validation.go

```diff
@@ -174,11 +171,7 @@ func validateVolumeAttachmentSource(source *storage.VolumeAttachmentSource, fldP
 	allErrs := field.ErrorList{}
 	switch {
 	case source.InlineVolumeSpec == nil && source.PersistentVolumeName == nil:
-		if utilfeature.DefaultFeatureGate.Enabled(features.CSIMigration) {
-			allErrs = append(allErrs, field.Required(fldPath, "must specify exactly one of inlineVolumeSpec and persistentVolumeName"))
-		} else {
-			allErrs = append(allErrs, field.Required(fldPath, "must specify persistentVolumeName when CSIMigration feature is disabled"))
-		}
+		allErrs = append(allErrs, field.Required(fldPath, "must specify exactly one of inlineVolumeSpec and persistentVolumeName"))
```

Comment:
Now that I am looking at this again, I wonder if an error message like: `must specify either inlineVolumeSpec or persistentVolumeName` makes more sense in the case where neither are specified. The `exactly one of ... ` is more appropriate for the case below (where both `persistentVolumeName` and `inlineVolumeSpec` are specified.)